samples/nrf9160/serial_lte_modem/prj.conf

```diff
@@ -40,14 +49,14 @@ CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=8192
 #CONFIG_AT_CMD_LOG_LEVEL_DBG=y
 
 # Device power management
-# Not ready for nRF91 in Zephyr yet
-#CONFIG_DEVICE_POWER_MANAGEMENT=y
+CONFIG_DEVICE_POWER_MANAGEMENT=y
 
 # Application-specific
 CONFIG_SLM_LOG_LEVEL_INF=y
-# Enable GPIO wakeup
+# Enable GPIO wakeup if sleep is expected
 #CONFIG_SLM_GPIO_WAKEUP=y
-# Use UART_2 when working with external MCU
+# Use UART_0 (when working with PC terminal)
+CONFIG_UART_0_NRF_HW_ASYNC_TIMER=2
+# Use UART_2 (when working with external MCU)
 #CONFIG_SLM_CONNECT_UART_2=y
-#CONFIG_UART_2_NRF_UARTE=y
-#CONFIG_UART_2_NRF_FLOW_CONTROL=y
+#CONFIG_UART_2_NRF_HW_ASYNC_TIMER=2
```

Comment:
Multiple UART instances cannot use the same timer.

Reply:
The project will use only one UART instance.